example/advection/advection_driver.cpp

```diff
@@ -136,6 +136,9 @@ TaskCollection AdvectionDriver::MakeTaskCollection(BlockList_t &blocks, const in
     auto update = tl.AddTask(avg_data, UpdateIndependentData<MeshData<Real>>, mc0.get(),
                              mdudt.get(), beta * dt, mc1.get());
 
+    // TODO(LFR): Remove this
+    update = tl.AddTask(update, advection_package::FillFine, mc1.get());
```

Comment:
still to be removed?

Reply:
Actually, I think just the comment should be removed since this at least tests fine fields a little bit. That being said, including the fine field slows down the advection tests some since the amount of output is substantially increased.